__tests__/app.test.js

```diff
@@ -0,0 +1,62 @@
+const app = require("../app");
+const request = require("supertest");
+const seed = require("../db/seeds/seed.js");
+const db = require("../db/connection");
+const data = require("../db/data/test-data");
+
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  db.end();
+});
+
+describe("GET api/categories", () => {
+  test("To return a status code of 200 and with an array of objects with a key of slug and a key of description", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body }) => {
+        body["categories"].forEach((catergory) => {
+          expect(catergory).toHaveProperty("slug");
+          expect(catergory).toHaveProperty("description");
+        });
+      });
+  });
+});
+
+describe("GET /api/reviews", () => {
+  test("To return the array of objects with all the properties and correct keys", () => {
+    return request(app)
+      .get("/api/reviews")
+      .expect(200)
+      .then(({ body }) => {
+        allReviews = body.reviews;
+        // console.log(allReviews)
+        allReviews.forEach((review) => {
+          expect(review).toHaveProperty("review_id", expect.any(Number));
+          expect(review).toHaveProperty("title", expect.any(String));
+          expect(review).toHaveProperty("designer", expect.any(String));
+          expect(review).toHaveProperty("owner", expect.any(String));
+          expect(review).toHaveProperty("review_img_url", expect.any(String));
+          expect(review).toHaveProperty("comment_count", expect.any(String));

```

Comment:
ideally this should be a number, look into how to cast the column data as a number instead. This can be done in your sql query